pages/pipelines/build_exports.md

```diff
@@ -1,12 +1,36 @@
-# Export build data
-
-Each [Buildkite plan](https://buildkite.com/pricing) has a maximum build retention
-period, after which your oldest builds are automatically deleted.
+# Build exports
 
 > 📘 Enterprise feature
-> Build export is only available on an [Enterprise](https://buildkite.com/pricing) plan.
+> Build exports is only available on an [Enterprise](https://buildkite.com/pricing) plan, which has a [build retention](/docs/pipelines/build-retention) period of 12 months.
+
+If you need to retain build data beyond the [retention period](/docs/pipelines/build-retention) in your [Buildkite plan](https://buildkite.com/pricing), you can use the build exports feature.
+
+Build exports enables you to configure your own S3 bucket for Buildkite to export your build data to.
```

Comment:
I'm unsure about referring to everything as a named feature, so let's avoid that here. I think we can also combine these two sentences:

```suggestion
If you need to retain build data beyond the [retention period](/docs/pipelines/build-retention) in your [Buildkite plan](https://buildkite.com/pricing), you can export the data to your own [Amazon S3 bucket](https://aws.amazon.com/s3/).
```

Note, I also referred to the full Amazon name and linked to it since this is the first mention.